src/sentry/static/sentry/app/views/eventsV2/table/columnEditCollection.tsx

```diff
@@ -45,14 +44,78 @@ enum PlaceholderPosition {
   BOTTOM,
 }
 
+export function generateFieldOptions(
```

Comment:
Not sure if this is the best place for it (@markstory?)

Reply:
This works for me.

Reply:
I would put this in a `table/utils` file -- I think re-usable utility functions should not be exported in a file that exports a component

Reply:
👍 to what billy's suggesting here.